games-rpg/runescape-launcher/runescape-launcher-2.2.4-r3.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit unpacker xdg gnome2-utils
+
+DESCRIPTION="Official RuneScape NXT client launcher"
+HOMEPAGE="http://www.runescape.com"
+
+SRC_URI="http://content.runescape.com/downloads/ubuntu/pool/non-free/r/${PN}/${PN}_${PV}_amd64.deb"
```

Comment:
Could you please use `->` to rename the file, and use a new name (e.g. append some revision or sth)?